src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/RequestSql/SqlRequestFormDataProvider.php

```diff
@@ -38,8 +38,18 @@
 use PrestaShop\PrestaShop\Core\Domain\SqlManagement\Query\GetSqlRequestForEditing;
 use PrestaShop\PrestaShop\Core\Domain\SqlManagement\ValueObject\SqlRequestId;
 
+@trigger_error(
+    sprintf(
+        '%s is deprecated since version 1.7.7.4 and will be removed in the next major version.',
```

Comment:
1.7.7.5

Reply:
I was thinkg it will be merged for the 1.7.7.4 :(